livemark/helpers.py

```diff
@@ -23,7 +24,7 @@ def read_asset(*paths):
 
 
 def get_relpath(path, current):
-    return os.path.relpath(path, os.path.dirname(current))
+    return posixpath.relpath(path, os.path.dirname(current))
```

Comment:
@shashigharti 
BTW it's a good way to solve the problem when we need "/" on windown we had similar situation with ZIP files if you remember. So we could have used this `posixpath` module